From 6cb24f02ca12c91d03dddc86c66e9befc5540cad Mon Sep 17 00:00:00 2001
From: Spencer Manning <spencerhmanning@gmail.com>
Date: Wed, 31 Jan 2024 22:05:46 -0500
Subject: [PATCH] Again remove all but scull and misc-modules

---
 Makefile | 5 +----
 1 file changed, 1 insertion(+), 4 deletions(-)

diff --git a/Makefile b/Makefile
index 7b08c49..eb6ed25 100644
--- a/Makefile
+++ b/Makefile
@@ -1,8 +1,5 @@
 
-SUBDIRS =  misc-progs misc-modules \
-           skull scull scullc scullp lddbus sculld scullv shortprint simple tty \
-	   pci usb\
-	   sbull snull short
+SUBDIRS = misc-modules scull
 
 all: subdirs
 
